locale/en/site.json

```diff
@@ -163,15 +163,10 @@
   },
   "banners": {
     "index": {
-      "startDate": "2022-01-04T16:00:00.000Z",
```

Comment:
@Trott as you're rebasing for the merge issue, I think you only need to change these 4 lines and the rest can be reverted